x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/action/XPackUsageResponseTests.java

```diff
@@ -26,17 +25,17 @@
 
 public class XPackUsageResponseTests extends ESTestCase {
 
-    private static Version oldVersion;
-    private static Version newVersion;
+    private static TransportVersion oldVersion;
+    private static TransportVersion newVersion;
 
     @BeforeClass
     public static void setVersion() {
-        oldVersion = VersionUtils.randomVersionBetween(
+        oldVersion = TransportVersionUtils.randomVersionBetween(
             random(),
-            VersionUtils.getFirstVersion(),
-            VersionUtils.getPreviousVersion(VersionUtils.getPreviousMinorVersion())
+            TransportVersionUtils.getFirstVersion(),
+            TransportVersionUtils.getPreviousVersion(TransportVersion.MINIMUM_COMPATIBLE)
```

Comment:
This is technically a behaviour change, as the concept of 'previous minor version' is going away - I think this is the closest applicable concept

Reply:
I don't think this is the same. The previous code meant to find a random version that is more than a minor version before the current version (not sure why though...). Using minimum compatible here is going to result in a much smaller window to choose from.

I would simplify this test to find a random version that does not include the current version, and then still choose the upgrade version to be between that and the current version (inclusive).

Reply:
Hmm, yeah, this now doesn't include versions between minimum compatible & current. I've updated it as suggested